modules/hcp-dc-hypervisor.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dc-hypervisor_{context}"]
+= Configuring the hypervisor for a disconnected installation of {hcp}
+
+To deploy {hcp} on a virtual platform in a disconnected environment, you need to configure a hypervisor.
```

Comment:
This whole procedure sounds completely misleading to me.
The kubevirt-hyperconverged operator (aka Openshift Virtualization) will take care of configuring the cluster nodes to be able to execute VMs as pods.
No need to manually configure any hypervisor.